src/current/cockroachcloud/network-authorization.md

```diff
@@ -1,45 +1,52 @@
 docs_area: manage
 ---
 
-{{ site.data.products.dedicated }} requires  explicit authorization of the networks that can access your cluster. This deny-by-default policy protects the cluster from network attacks including denial-of-service and password brute-forcing.
+{{ site.data.products.dedicated }} allows explicit configurations of the networks that can access your cluster.
 
-You can do this by:
+To prevent denial-of-service attacks, brute force password attacks, and other forms of malicious activity, Cockroach Labs recommends restricting your network to allow access only from specific IP address ranges controlled by your organization, according to the [Principle of Least Privilege (PoLP)](https://en.wikipedia.org/wiki/Principle_of_least_privilege). These might include specific networks for your application servers, hardened administrator access points, and backup-restore pipelines for disaster recovery.
 
-- [Adding an authorized range of public IP addresses](#ip-allowlisting).
-- Setting up [Google Cloud Platform (GCP) Virtual Private Cloud (VPC) peering](#vpc-peering) or [Amazon Web Service (AWS) PrivateLink](#aws-privatelink) for your cluster.
+## Options for controlling network access
 
-**Prerequisite**: {% include cockroachcloud/cluster-operator-prereq.md %}
+You can authorize network access to your cluster by:
 
-{{site.data.alerts.callout_success}}
-For additional cluster security, you can learn more about [Private Clusters (Preview)](private-clusters.html). A private cluster's nodes have no public IP addresses.
+- [Adding an authorized range of public IP addresses](#ip-allowlisting).
+- Setting up [Google Cloud Platform (GCP) Virtual Private Cloud (VPC) peering](#vpc-peering) or [Amazon Web Service (AWS) PrivateLink](#aws-privatelink) for your cluster (Dedicated clusters only). Access via [GCP VPC peering](#vpc-peering) or [AWS PrivateLink](#aws-privatelink) avoids traversing the public internet, and therefore offers several advantages:
 
-You should use PrivateLink or VPC peering if you need to allowlist more than 20 IP addresses, if your servers’ IP addresses are not static, or if you want to limit your cluster's exposure to the public internet.
-{{site.data.alerts.end}}
+    - Enhanced network security (no public IPs, no transit over public networks).
+    - Direct connection to application servers that do not have static public IPs
+    - Reduced lower network latency.
+    - This option can also be helpful if you need more than the current maximum of 20 authorized networks per cluster.
 
 {{site.data.alerts.callout_info}}
+
 During [limited access](/docs/{{site.versions["stable"]}}/cockroachdb-feature-availability.html), neither Azure Private Link nor private clusters are available for {{ site.data.products.dedicated }} clusters on Azure. Refer to [{{ site.data.products.dedicated }} on Azure](cockroachdb-dedicated-on-azure.html).
 {{site.data.alerts.end}}
 
-## IP allowlisting
+## Cluster default network configuration
```

Comment:
Should this go as a sub-section in IP allowlisting?

Reply:
no, because the 'lockdown' nature of dedicated clusters on creation includes other kinds of networking as well.